include/fmt/format.h

```diff
@@ -150,16 +150,16 @@ FMT_END_NAMESPACE
 
 // __builtin_clz is broken in clang with Microsoft CodeGen:
 // https://github.com/fmtlib/fmt/issues/519
-#if (FMT_GCC_VERSION || FMT_HAS_BUILTIN(__builtin_clz)) && !FMT_MSC_VER
+#if (FMT_GCC_VERSION || FMT_ICC_VERSION || FMT_HAS_BUILTIN(__builtin_clz)) && !FMT_MSC_VER
```

Comment:
LGTM but could you move the ICC check after `FMT_HAS_BUILTIN`? Same below.

```c++
#if (FMT_GCC_VERSION || FMT_HAS_BUILTIN(__builtin_clz) || FMT_ICC_VERSION) && !FMT_MSC_VER
```

Reply:
done.